components/script/dom/window.rs

```diff
@@ -330,6 +336,29 @@ impl Window {
     pub fn webvr_thread(&self) -> Option<IpcSender<WebVRMsg>> {
         self.webvr_thread.clone()
     }
+
+    #[allow(unrooted_must_root)]
```

Comment:
`Entry` is like `Ref`, please add it to the list of exceptions and remove the `allow`.